pkg/webhook/work/mutating.go

```diff
@@ -55,49 +53,15 @@ func (a *MutatingAdmission) Handle(_ context.Context, req admission.Request) adm
 		util.MergeLabel(work, workv1alpha2.WorkPermanentIDLabel, uuid.New().String())
 	}
 
-	var manifests []workv1alpha1.Manifest
-
-	for _, manifest := range work.Spec.Workload.Manifests {
-		workloadObj := &unstructured.Unstructured{}
-		err := json.Unmarshal(manifest.Raw, workloadObj)
-		if err != nil {
-			klog.Errorf("Failed to unmarshal the work(%s/%s) manifest to Unstructured, err: %v", work.Namespace, work.Name, err)
-			return admission.Errored(http.StatusInternalServerError, err)
-		}
-
-		err = prune.RemoveIrrelevantFields(workloadObj, prune.RemoveJobTTLSeconds)
-		if err != nil {
-			klog.Errorf("Failed to remove irrelevant fields for the work(%s/%s), err: %v", work.Namespace, work.Name, err)
-			return admission.Errored(http.StatusInternalServerError, err)
-		}
-
-		// Skip label/annotate the workload of Work that is not intended to be propagated.
-		if work.Labels[util.PropagationInstruction] != util.PropagationInstructionSuppressed {
-			setLabelsAndAnnotationsForWorkload(workloadObj, work)
-		}
```

Comment:
@XiShanYongYe-Chang @whitewindmills Do you remember why we skipped the step of setting the label?

Reply:
For resources collected from member clusters, such as `EndpointSlice` resources in the MCS feature, the work is not propagated. Therefore, we do not need to set labels and annotations.